src/ProxyManager/ProxyGenerator/Util/PublicScopeSimulator.php

```diff
@@ -108,8 +108,8 @@ private static function getUndefinedPropertyNotice(string $operationType, string
             . '            \'Undefined property: %s::$%s in %s on line %s\',' . "\n"
             . '            get_parent_class($this),' . "\n"
             . '            $' . $nameParameter . ',' . "\n"
-            . '            $backtrace[0][\'file\'],' . "\n"
-            . '            $backtrace[0][\'line\']' . "\n"
+            . '            $backtrace[0][\'file\'] ?? \'\',' . "\n"
```

Comment:
Code like this:

`
class Foo {
    private $value = 'Value';
}

class Bar  extends Foo{
}

$factory = new \ProxyManager\Factory\LazyLoadingValueHolderFactory();
$proxy = $factory->createProxy(
    \Bar::class,
    function (& $wrappedObject, $proxy, $method, $parameters, & $initializer) {
        $wrappedObject = new Bar(); // instantiation logic here
        $initializer   = null; // turning off further lazy initialization
    }
);

$r = new \ReflectionClass(\Foo::class);
$p = $r->getProperty('value');
$p->setAccessible(true);

$p->getValue($proxy);
`

causes output:

PHP Notice:  Undefined index: file in /vendor/ocramius/proxy-manager/src/ProxyManager/GeneratorStrategy/EvaluatingGeneratorStrategy.php(70) : eval()'d code on line 85

PHP Notice:  Undefined index: line in /vendor/ocramius/proxy-manager/src/ProxyManager/GeneratorStrategy/EvaluatingGeneratorStrategy.php(70) : eval()'d code on line 86

PHP Notice:  Undefined property: Bar::$value in  on line  in /vendor/ocramius/proxy-manager/src/ProxyManager/GeneratorStrategy/EvaluatingGeneratorStrategy.php(70) : eval()'d code on line 88